<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>310</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>310</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>251</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Komunikacja z plikiem</string>
    </property>
    <widget class="QLabel" name="pathDisp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>751</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>brak pliku</string>
     </property>
    </widget>
    <widget class="QPushButton" name="writeBut">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>150</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Zapisz całosć</string>
     </property>
    </widget>
    <widget class="QPushButton" name="readBut">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>100</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Odczytaj całość</string>
     </property>
    </widget>
    <widget class="QPushButton" name="selectNewFile">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>50</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Wskaż plik</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="textDisp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>451</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="clearBut">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>200</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Wyczyść plik</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addValueBut">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Dodaj wartość</string>
     </property>
    </widget>
    <widget class="QPushButton" name="getValueBut">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>200</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Odczytaj wartość</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="readBox">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>170</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="setVal">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wartość do zapisu:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Wartość odczytana:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="revBut">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Przewiń</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
